python/ray/experimental/state.py

```diff
@@ -594,31 +626,141 @@ def micros_rel(ts):
                         "cat": "execute",
                         "pid": "Node " + str(worker["node_ip_address"]),
                         "tid": info["worker_id"],
-                        "id": str(worker),
+                        "id": str(task_id),
                         "ts": micros_rel(info["execute_start"]),
                         "ph": "X",
                         "name": info["function_name"] + ":execute",
-                        "args": delta_info,
+                        "args": total_info,
                         "dur": micros(info["execute_end"] -
-                                      info["execute_start"])
+                                      info["execute_start"]),
+                        "cname": "rail_animation"
```

Comment:
it's a color. they have a list of reserved color names that the colors of the edges can be changed to. this is one of them